### DISCUSSION TITLE
How to use NSFW filter

### DISCUSSION THREAD

**Discussion by canytam-krystal** [lllyasviel/Fooocus#3452] [Q&A] [answered]
As I heard that there is a NSFW filter, but I don't know how to use it?

**Comment by oldhand7** [comment]
Yes, you can download from github 
https://github.com/nsfw-filter/nsfw-filter
You can use this.
Easy to implement.

**Comment by lgwjames** [comment]
or use the nsfw etc in the neg prompt

**Comment by mashb1t** [comment, marked as answer]
@canytam-krystal please find the checkbox for `Black Out NSFW` in advanced > advanced > developer debug mode. To enable it simply set it to true. This can also be enabled by default using the config file.
